MEMO-Zi/app/build.gradle

```
@@ -41,6 +41,14 @@ android {
 
 dependencies {
 
+    implementation "com.kakao.sdk:v2-all:2.18.0" // 전체 모듈 설치, 2.11.0 버전부터 지원
+    implementation "com.kakao.sdk:v2-user:2.18.0" // 카카오 로그인
+    implementation "com.kakao.sdk:v2-talk:2.18.0" // 친구, 메시지(카카오톡)
+    implementation "com.kakao.sdk:v2-share:2.18.0" // 메시지(카카오톡 공유)
+    implementation "com.kakao.sdk:v2-friend:2.18.0" // 카카오톡 소셜 피커, 리소스 번들 파일 포함
+    implementation "com.kakao.sdk:v2-navi:2.18.0" // 카카오내비
+    implementation "com.kakao.sdk:v2-cert:2.18.0" // 카카오 인증서비스
```

```suggestion
    // kakao 로그인
    implementation "com.kakao.sdk:v2-all:2.18.0"
    implementation "com.kakao.sdk:v2-user:2.18.0"
    implementation "com.kakao.sdk:v2-talk:2.18.0"
    implementation "com.kakao.sdk:v2-share:2.18.0"
    implementation "com.kakao.sdk:v2-friend:2.18.0"
    implementation "com.kakao.sdk:v2-navi:2.18.0"
    implementation "com.kakao.sdk:v2-cert:2.18.0"
```
이런식으로만 작성해주셔도 좋을 것 같아요